File: PHP/1006.php
<?php
    $a = trim(fgets(STDIN));
    $b = trim(fgets(STDIN));
    $c = trim(fgets(STDIN));
    printf("MEDIA = %.1f\n", ($a * 2 + $b * 3 + $c * 5) / 10);
?>

File: Rust/1006.rs
use std::io;

fn main() {
    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let a: f64 = input.trim().parse().unwrap();

    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let b: f64 = input.trim().parse().unwrap();

    let mut input = String::new();
    io::stdin().read_line(&mut input);
    let c: f64 = input.trim().parse().unwrap();

    println!("MEDIA = {:.1}", (a * 2.0 + b * 3.0 + c * 5.0) / 10.0);
}